portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTimeFilter.macro

```diff
@@ -48,13 +48,15 @@ definition {
 	macro setCustomRange {
 		var key_dropdownOption = "Custom Range";
 
+		Click(locator1 = "ACTimeFilter#TIME_FILTER_DROPDOWN");
```

Comment:
This doesn't seem necessary since there's a macro that will click time filter dropdown. https://github.com/calvinctvo/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTimeFilter.macro#L30-L39